### ISSUE TITLE
Scroll now shown when we have multiple farms

### ISSUE THREAD

**Issue by mydesravines** [nazar-pc/space-acres#73]
Scroll feature from the GUI not shown or doesn't exist when we have multiple farms to go up and down. 

<img width="279" alt="Screenshot 2024-01-03 161208" src="https://github.com/nazar-pc/space-acres/assets/81017315/4e2e1582-609c-48cc-acdc-3d2a0dd06ab4">

In my case, I had to change my screen resolution to be able to see the SAVE or Cancel button.

**Comment by nazar-pc** [comment]
I can't reproduce this, both on Windows (10 and 11) and Linux (Ubuntu 22.04) view is scrolling just fine

**Comment by mydesravines** [comment]
I tested again. With a 100% or 125% screen resolution, when I go to add more farms, I can see all the GUI and Scroll is working well as you said. 

My problem was at 150% as I had it initially with 9 farmers.SO when I go to edit the configuration, I could not see the botton of the GUI with the Save or Cancel button.  That's why I though the scroll was not working.

So I had to change de resolution. Adding more farmers, scroll works perfectly. 

Thanks.
